_includes/pbc/all/install-features/202400.0/install-the-shipment-feature.md

```diff
@@ -107,7 +108,34 @@ class GlueBackendApiApplicationAuthorizationConnectorConfig extends SprykerGlueB
 }
 ```
 
-### 3) Set up database schema and transfer objects
+### 3) To enable the Storefront API, register the following plugins:
+| PLUGIN                           | SPECIFICATION                            | PREREQUISITES | NAMESPACE                                                |
+|----------------------------------|------------------------------------------|---------------|----------------------------------------------------------|
+| ShipmentTypesResourceRoutePlugin | Registers the `shipment-types` resource. |               | Spryker\Glue\ShipmentTypesRestApi\Plugin\GlueApplication |
+
+**src/Pyz/Glue/GlueApplication/GlueApplicationDependencyProvider.php**
+```php
+<?php
+
+namespace Pyz\Glue\GlueApplication;
+
+use Spryker\Glue\GlueApplication\GlueApplicationDependencyProvider as SprykerGlueApplicationDependencyProvider;
+use Spryker\Glue\ShipmentTypesRestApi\Plugin\GlueApplication\ShipmentTypesResourceRoutePlugin;
+
+class GlueApplicationDependencyProvider extends SprykerGlueApplicationDependencyProvider
+{
+    /**
+     * {@inheritDoc}
+     *
+     * @return array<\Spryker\Glue\GlueApplicationExtension\Dependency\Plugin\ResourceRoutePluginInterface>
+     */
+    protected function getResourceRoutePlugins(): array
+    {
+        new ShipmentTypesResourceRoutePlugin(),
+    }
+}
+```
```

Comment:
Please add a verify block to ensure the route plugin works as expected.

Reply:
done. Nice catch :)